packages/costa/src/runtime.ts

```diff
@@ -271,9 +294,9 @@ export class CostaRuntime {
     }
     if (!await pathExists(`${this.options.installDir}/package.json`)) {
       // if not init for plugin directory, just run `npm init` and install boa firstly.
-      await spawnAsync('npm', [ 'init', '-y' ], npmExecOpts);
+      await spawnAsync('npm', [ 'init', '-y' ], npmExecOpts, logWriteStream);
```

Comment:
How about moving the `spawnAsync()` as the class method, so that the `logWriteStream` should be accessed via `this`.

Reply:
The `logWriteStream` is request-scope, if we put it on `this`, it will be confusion by different requests.